### PR TITLE
[ADD] Added access rights and rules of proposal model for Portal group

### PR DESCRIPTION
**Pull request by krunal267**
Task URL:

https://www.druidoo.io/web#id=1844&model=project.task&view_type=form&menu_id=237

### CODE REVIEW

File: intercoop_addons/coop_memberspace/security/ir_rule.xml [hunk 1]
```diff
@@ -11,5 +11,13 @@
...
             <field name="groups" eval="[(4, ref('coop_memberspace.group_memberspace'))]"/>
         </record>
+
+        <record id="member_space_my_proposals" model="ir.rule">
+            <field name="name">MemberSpace My Proposals</field>
+            <field name="model_id" ref="coop_memberspace.model_proposal"/>
+            <field name="domain_force">['|', ('src_registration_id.partner_id', '=', user.partner_id.id), ('des_registration_id.partner_id', '=', user.partner_id.id)]</field>
+            <field name="groups" eval="[(4, ref('base.group_portal'))]"/>
+        </record>
+
```
Comment (ivantodorovich) [inline intercoop_addons/coop_memberspace/security/ir_rule.xml:21]: Don't we need a [(1,'=',1)] rule for manager user groups?

I'm not familiar with this module but I don't want to break it while trying to fix it ^^

**Review by ivantodorovich — CHANGES_REQUESTED** [review]
Regarding the use of sudo().

I'm answring your comment here:

> I have added access rights and rules of proposal model for Portal group as proposal record's state field is getting changed while cancelling it, so if we use sudo() then it would become difficult to track who changed the record.

Not sure I understand, which fields would we be losing?
Is it only the create_uid/write_uid, or are you worried about some others?

We can use sudo() and then write on the fields you're worried about, if tht's an issue.


---

Anyways, any approach is ok to me. But if we go for security rules please make sure everything works as expected. ^^

File: intercoop_addons/coop_memberspace/security/ir_rule.xml (hunk 1 above)
Comment (ivantodorovich) [reply intercoop_addons/coop_memberspace/security/ir_rule.xml:21]: @krunal267 No hurry but please revert on this pls so we can close it :)

**Comment by msdruidoo** [comment]
@krunal267 
could you update on this one please?

**Pull request closed without merging**